DEVELOPMENT.md

```diff
@@ -0,0 +1,18 @@
+# Development
+
+This repository contains the scripts and patches used to build and download the Node.js binaries for pkg.
+
+## New releases
+
+Pkg will check for releases of pkg-fetch `<major>.<minor>` version. So each minor/major release of pkg-fetch requires a new GH release with all attached binaries. Patch releases don't require to recreate all new binaries, but only the ones that have changed.
+
+Important things:
+
+- In order to support a new nodejs version we need to create/update the patch in `patches` folder and also `patches/patches.json` file.
+- Once the new patch is merged we can trigger the `.github/workflows/build-all.yml` workflow to build all the binaries for all the platforms.
+- If this is a minor/major release we need to check the checkbox of the workflow to create a draft release.
+- Once the actions will end, copy the sha256 checksums printed at the end of the release body and update`lib/expected-shas.json` file with the correct values.
+- If this is not a new release, download the new nodejs binaries and upload them as assets to the active release.
```

Comment:
I think there was one part of the build-all workflow that automatically uploaded the binaries to the latest release or something, but I'd have to dig into the workflow to find out

Reply:
I thought too but there is only a checkbox to check when triggering the workflow that is used to draft a new release. In fact now the new release has been created: https://github.com/yao-pkg/pkg-fetch/releases/tag/untagged-e45775aacda181c94894

Dunno now if I should simply delete the previous one or what else?

P.S: the collect artifact was failing as it expected a release to exists, I had to create one manually (copy paste of the one in original repo)

Reply:
I have deleted the previous "fake" release and renamed the untagged one with the correct changelog and so on

Reply:
Yeah when the checkbox `Create new draft release` of the build-all workflow is not checked it will upload releases to the latest one, problem is that this is a fork and there is no releases so I had to create a fake one first and then I have deleted the "fake" release and renamed the untagged one created by the workflow with the correct changelog. All artifacts have been built successfully 🎉